pkgs/top-level/python-packages.nix

```diff
@@ -155,6 +155,8 @@ in modules // {
 
   bugseverywhere = callPackage ../applications/version-management/bugseverywhere {};
 
+  buildbot-plugins = callPackage ../development/tools/build-managers/buildbot/plugins.nix { };
```

Comment:
@FRidh not clear what you mean.  buildbot-plugins is in python-packages here, as it would not make sense for it to be in top-level all-packages?